dirsrvtests/tests/suites/acl/keywords_test.py

```diff
@@ -223,7 +223,8 @@ def test_user_binds_without_any_password_and_cannot_access_the_data(topo, add_us
     with pytest.raises(ldap.INSUFFICIENT_ACCESS):
         org.replace("seeAlso", "cn=1")
 
-
+#unstable or unstatus tests, skipped for now
+@pytest.mark.xfail(raises=ldap.INSUFFICIENT_ACCESS)
```

Comment:
When a test marked with `xfail` passes, it is marked as `xpass` (unexpectedly passing). `xpass` is interpreted by some systems as a failure (because it's unexpected and needs investigation).
Instead of `xfail` I suggest to mark these tests as `@pytest.mark.flaky`. This way they can be skipped in PR CI by excluding all tests with this mark (`pytest -m "not flaky`). 
Additionally we can take advantage of [pytest flaky plugin](https://github.com/box/flaky), that can automatically rerun flaky tests on failure.

Reply:
PR CI pytest command can be extended with the mark selector in https://github.com/389ds/389-ds-base/blob/4559a89c0877e45143fa3c9cc34393d4d0cd2696/.github/workflows/pytest.yml#L68

Reply:
Also, looking at the failing tests, they mostly related to IPv6 connectivity. I will investigate what configuration changes we need to do in GH Actions.